### ISSUE TITLE
You can deep fry pets endlessly 

### ISSUE THREAD

**Issue by Moonman121** [tgstation/tgstation#50524]
## Round ID:
134990

Steps:
1. pick up ian or any other pet
2. place them in a deep fryer 
3. cook until golden brown, take them out and bite them
4. ???

![Untitled](https://user-images.githubusercontent.com/43108044/79627351-d0f60a00-817a-11ea-8f2b-181b50d9546c.png)

**Comment by ATH1909** [comment]
Does it stack up the "deep-fried" prefixes?

**Comment by ArcaneDefence** [comment]
No, it's just that when you bite the reagent container placeholder item it releases any mob inside of it. Since the fried item isn't eaten in one bite it remains. 
This works even if you only lightly fry it, but then inject enough reagents that it's not consumed in one bite.

For every placeholder item you have to pick up the pet again.

**Comment by NanoCats** [comment]
aw man I was hoping this would get left in but you've gone and ruined the fun, now it's gonna get fixed. I always thought it was hilarious to grab Ian, deep fry him, make the HOP think I was a complete monster, and then show him alive-ian.

**Comment by optimumtact** [comment]
Now I have an image of you biting in to an ian sized donut fry and a real corgi bursts out and yaps in excitement and then dives back into the deepfryer

**Comment by Shadowflame909** [comment]
this bug still exists lol. Infinite food source

**Comment by Sealed101** [comment]
#71551 was implemented some time ago, could have resolved this
